src/main/java/org/opentripplanner/routing/graph/Graph.java

```diff
@@ -117,6 +118,11 @@ public class Graph implements Serializable {
   public DataOverlayParameterBindings dataOverlayParameterBindings;
   private LuceneIndex luceneIndex;
 
+  /**
+   * Issues encountered during graph build summarized by type.
+   */
+  private Map<String, Long> issueSummary = Map.of();
```

Comment:
I think this should be encapsulated into a container class and then put as a separate field on the `SerializedGraphObject` we should try to remove stuff from this class, not add to it. Also, by adding it as a separate field in the `SerializedGraphObject` we may be able to support larger graphs (reset ref index between fields) as well as adding flags for turning the serialization on/off. 

Is there any reason to keep this in the graph?

Reply:
The `SummarizeDataImportIssues` can be refactored to be the container of the map and made serializable...

Reply:
I agree that it's better to not use verbs in class names.